src/pages/api/member/[member].ts

```diff
@@ -0,0 +1,12 @@
+import type {GetServerSideProps, NextApiRequest, NextApiResponse} from 'next'
+import { prisma } from "../../../server/db/client";
+
+export const getServerSideProps = async (context: { params: { studentId: string; }; }) => {
```

Comment:
May I trouble you to shift this segment of the code to the TRPC? You may take reference from the /server/router/example.ts and merge it in to the index.ts. The same applies for the remaining 4 files